pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -450,6 +494,19 @@ func (nc *Controller) doFixDeprecatedTaintKeyPass(node *v1.Node) error {
 	return nil
 }
 
+func (nc *Controller) doNoScheduleTaintingPassWorker(i int, stopCh <-chan struct{}) {
+	for {
+		select {
+		case <-stopCh:
+			break
```

Comment:
IMO `break` will cause _slight_ goroutine (entity of running this function) leak here.. As `break` only breaks out of the `select` loop.

Changing to `return` would be better.

Reply:
yes, return is better :)